info/GameTestList.md

```diff
@@ -0,0 +1,21 @@
+# Games with Goose
+**Notice:** At the moment there is someone testing games to see if the games will ban player while having the goose in the background
+
+Some games may ban the players while the desktop goose is being played in the background.
+
+| Symbol | Meaning                        |
+| ------ |:------------------------------:|
+| ✔️    | Okay to play the goose          |
+| ❌    | Will get banned using the goose |
+
+## List of Tested Games
+
+* CS:GO ✔️ 
```

Comment:
![image](https://user-images.githubusercontent.com/46850780/73684090-c4a5f900-46c3-11ea-894f-b4a1a38485ee.png)
You could use a table for the games as well (obvious mockup with inspect element here so didn't change the last one, you get the point anyway)

Reply:
Oh, I'll change it up then for better formatting.